github/ci/prow-deploy/files/jobs/kubevirt/cloud-provider-kubevirt/cloud-provider-kubevirt-presubmits.yaml

```diff
@@ -38,3 +38,36 @@ presubmits:
           resources:
             requests:
               memory: "10Gi"
+  - name: pull-kubevirt-cloud-provider-kubevirt-e2e
+    always_run: true
+    optional: true
+    decorate: true
+    cluster: prow-workloads
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 3h0m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+    max_concurrency: 11
+    skip_branches:
+    - release-\d+\.\d+
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - make cluster-up cluster-sync functest
```

Comment:
Please run here only one target `make functest`
And trigger all dependencies from the repository itself
The same like done here: https://github.com/kubevirt/project-infra/blob/9c822fb8cdb45da0760aaeed2701c33b214826b3/github/ci/prow-deploy/files/jobs/kubevirt/cluster-api-provider-kubevirt/cluster-api-provier-kubevirt-presubmits.yaml#L22

Reply:
Done